src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+
+```js
+const openAnimation = [
+  { transform: 'scale(0)' }, 
+  { transform: 'scale(1)' }, 
+];
+```
+
+Where you set parameters for animation in CSS:
+
+```css
+.modal {
+  animation: openAnimation 1s 1 ease-in;
+}
+```
+
+You would set in JS:
+
+```js
+document.querySelector('.modal').animate(
+    openAnimation ,  {
+    duration: 1000, // 1s
+    iterations: 1, // single iteration
+    easing: 'ease-in' // easing function
+  }
+);
+```
+
+The amount of code is about the same, but with JavaScript, you get a couple of superpowers that you don't have with CSS alone. This includes the ability to sequence effects, and an increased control of their play states.
+
+{% Aside %}
+  Hyphenated property names become camel case when used in keyframes (i.e. `"background-color"` to `"backgroundColor"`)
+{% endAside %}
+
+## Orchestrating animations with promises
+
+In Chrome 84, you now have two methods that can be used with promises: `"animation.ready"` and `"animation.finished"`.
+
+- `animation.ready` enables you to wait for pending changes to take effect (i.e. switching between playback control methods such as play and pause).
+- `animation.finished` provides a means of executing custom JavaScript code when an animation is complete.
+
+Let's continue with our example, and create an orchestrated animation chain with `"animation.finished"`. Here, you have a vertical transformation (`"scaleY"`), followed by a horizontal transformation (`"scaleX"`), followed by an opacity change on a child element: 
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/modal-open.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Applying transformations and opacity to an opening modal element. <a href="https://codepen.io/una/pen/dyYKJMz">See Demo on Codepen</a>
+  </figcaption>
+</figure>
+
+```js/1
+const transformAnimation = modal.animate(openModal, openModalSettings);
+transformAnimation.finished.then(() => { text.animate(fadeIn, fadeInSettings)});
+```
+
+We've chained these animations using `"animation.finished.then()"` prior to executing the next animation set in the chain. This way, the animations appear in order, and you are even applying effects to different target elements with different options set (such as speed and ease).
+
+Within CSS, this would be cumbersome to recreate, especially when applying unique, yet sequenced animations to multiple elements. You'd have to use a `"@keyframe"`, sort out the correct timing percentages to place the animations, and use `"animation-delay"` prior to triggering the animations in the sequence.
+
+## Play, pause, and reverse
+
+What can open, should close! Luckily, since [Chrome 39](https://developers.google.com/web/updates/2014/12/web-animation-playback), the WAAPI has provided us the ability to play, pause, and reverse our animations.
```

Comment:
```suggestion
What can open, should close! Luckily, since [Chromium 39](https://developers.google.com/web/updates/2014/12/web-animation-playback), the Web Animations API has provided us the ability to play, pause, and reverse our animations.
```
We haven't defined the acronym `WAAPI` anywhere else in this article and it looks weird to me so I think it'd be safer to just use the full name throughout